### PR TITLE
Currency: Added build-script and build_info for apache-httpd.

### PR DESCRIPTION
**Pull request by AbhiIBM**
Currency: Added build-script and build_info for httpd.

## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

**Review by nikhil-kalbande — CHANGES_REQUESTED** [review]
@AbhiIBM  this is apache package. So lets add this as apache-httpd. Do other required changes to handle that

**Comment by AbhiIBM** [comment]
Updated the names of all the scripts as per naming convention.

File: a/apache-httpd/build_info.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    "maintainer":"Abhishek.Dwivedi6@ibm.com",
+    "package_name" : "httpd",
```
Comment (nikhil-kalbande) [inline a/apache-httpd/build_info.json:3]: here it should be "apache-httpd"

**Review by nikhil-kalbande — CHANGES_REQUESTED** [review]

File: a/apache-httpd/build_info.json (hunk 1 above)
Comment (AbhiIBM) [reply a/apache-httpd/build_info.json]: Updated the package name to "apache-httpd"

**Review by nikhil-kalbande — APPROVED** [review]

**Pull request merged**